salt/docker_apps/files/spoonbill.yaml

```diff
@@ -0,0 +1,105 @@
+version: '3.7'
+
+x-base: &base
+  env_file: .env
+  restart: unless-stopped
+  user: "{{ pillar.docker.uid }}:{{ pillar.docker.uid }}"
+
+x-shared: &shared
+  <<: *base
+  image: "ghcr.io/open-contracting/spoonbill-web/app:latest"
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
+  volumes:
+    # The right-hand path must match the settings.FILE_UPLOAD_TEMP_DIR default value in spoonbill-web.
+    - /data/storage/spoonbill/tmp:/data/tmp
+    # The right-hand path must match the settings.MEDIA_ROOT default value in spoonbill-web.
+    - /data/storage/spoonbill/media:/data/media
+    # The right-hand path must match the settings.DATAREGISTRY_MEDIA_ROOT default value in spoonbill-web.
+    - {{ pillar.docker_apps.registry.env.EXPORTER_HOST_DIR }}:/data/exporter
+
+services:
+  traefik:
+    <<: *base
+    image: traefik:v2.4
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./:/letsencrypt
+    ports:
+      - ${TRAEFIK_IP:-0.0.0.0}:80:80
+      - ${TRAEFIK_IP:-0.0.0.0}:443:443
+    command:
+      - '--api'
+      # - '--api.insecure=true'
+      - '--providers.docker=true'
+      - '--providers.docker.exposedByDefault=false'
+      - '--entrypoints.http=true'
+      - '--entrypoints.http.address=:80'
+      - '--entrypoints.http.http.redirections.entrypoint.to=https'
+      - '--entrypoints.http.http.redirections.entrypoint.scheme=https'
+      - '--entrypoints.https=true'
+      - '--entrypoints.https.address=:443'
+      - '--log=true'
+      - '--log.level=INFO'
+      - '--certificatesresolvers.acmec.acme.email=${ACME_EMAIL}'
```

Comment:
You can substitute the value from Pillar here. In future it will be sysadmin@ and never change.